neuralprophet/forecaster.py

```diff
@@ -1223,7 +1283,13 @@ def _predict_raw(self, df, include_components=False):
                             component_vectors[name].append(value.detach().numpy())
 
         predicted = np.concatenate(predicted_vectors)
-        scale_y, shift_y = self.data_params["y"].scale, self.data_params["y"].shift
+        if self.local_modeling:
+            scale_y, shift_y = (
+                self.data_params.norm_params_dict[df_names]["y"].scale,
```

Comment:
beware - indexing with a list. this will likely not work.
We likely need to call predict_raw len(df_names) times and return it as a list/dict

Reply:
or is it a type and it should not be a list but only a single name?

Reply:
It is a typo, I changed for "df_name". _predict_raw is always called inside a loop in which we go over each dataframe name each time.